### ISSUE TITLE
不会gradle构建项目，有没有使用maven构建的sample

### ISSUE THREAD

**Issue by ChadDongDongChen** [spring-projects/spring-security-samples#126]

**Comment by marcusdacoregio** [comment]
Hi @ChadDongDongChen, we use English as the default language for communication, are you able to translate the issue title and provide some context?

**Comment by DevDengChao** [comment]
@ChadDongDongChen Well, you can learn how to use Gradle to build a project here: https://gradle.org/ . 🤣

**Comment by BingChunMoLi** [comment]
You should make good use of search engines
Find code from [gitee.com](https://gitee.com)

**Comment by horaoen** [comment]
I don't think you need to master gradle, You only need to configure the gradle environment to build the project, and the Chinese network needs to replace the mirror source such as the aliyun's gradle mirror [阿里云maven repo代理文档](https://help.aliyun.com/document_detail/102512.html)
